### PR TITLE
Create ui-c8y-1021-3-0-get-context-dashboards-fix-7570-graft-release-cd-7573.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-61691](https://cumulocity.atlassian.net/browse/MTM-61691)
Original PR: [7573](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7573)

[MTM-61691]: https://cumulocity.atlassian.net/browse/MTM-61691?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-3-0-get-context-dashboards-fix-7570-graft-release-cd-7573.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Get context dashboards fix (#7570) [GRAFT][release/cd] (#7573)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-3-0-get-context-dashboards-fix-7570-graft-release-cd-7573.md]: The Claude AI suggested to use the following title:
```suggestion
title: Fix retrieving context dashboards
```

File: content/change-logs/application-enablement/ui-c8y-1021-3-0-get-context-dashboards-fix-7570-graft-release-cd-7573.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61691
+version: 1021.3.0
+---
+Get context dashboards fix (#7570) [GRAFT][release/cd] (#7573)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-3-0-get-context-dashboards-fix-7570-graft-release-cd-7573.md]: The Claude AI suggested to use the following body:
```suggestion
In certain situations, retrieving context dashboards did not work as expected. This has now been fixed. Context dashboards can now be retrieved reliably in all cases. This fix ensures that users can always access and view their context dashboards as intended.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1021-3-0-get-context-dashboards-fix-7570-graft-release-cd-7573.md (hunk 1 above)
Comment (jdre-c8y) [inline content/change-logs/application-enablement/ui-c8y-1021-3-0-get-context-dashboards-fix-7570-graft-release-cd-7573.md]:
```suggestion
title: Fix retrieving device context dashboards
```

File: content/change-logs/application-enablement/ui-c8y-1021-3-0-get-context-dashboards-fix-7570-graft-release-cd-7573.md (hunk 2 above)
Comment (jdre-c8y) [inline content/change-logs/application-enablement/ui-c8y-1021-3-0-get-context-dashboards-fix-7570-graft-release-cd-7573.md]: Retrieving device context dashboards did not work as expected if device had space in it's name, so it was impossible to view device view as dashboard is by default first route for device in Cockpit. This has now been fixed.

Comment (jdre-c8y) [inline content/change-logs/application-enablement/ui-c8y-1021-3-0-get-context-dashboards-fix-7570-graft-release-cd-7573.md]:
```suggestion
Retrieving device context dashboards did not work as expected if device had space in it's name, so it was impossible to view device view as dashboard is by default first route for device in Cockpit. This has now been fixed.
```

File: content/change-logs/application-enablement/ui-c8y-1021-3-0-get-context-dashboards-fix-7570-graft-release-cd-7573.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61691
+version: 1021.3.0
+---
+Retrieving device context dashboards did not work as expected if device had space in it's name, so it was impossible to view device view as dashboard is by default first route for device in Cockpit. This has now been fixed.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-3-0-get-context-dashboards-fix-7570-graft-release-cd-7573.md]:
```suggestion
Retrieving the device context dashboards did not work as expected if the device had a space in its name. As a result, it was impossible to view such devices as the dashboard is by default the initial view for devices in the Cockpit application. This issue has been fixed and the device dashboard is always displayed properly.
```

File: content/change-logs/application-enablement/ui-c8y-1021-3-0-get-context-dashboards-fix-7570-graft-release-cd-7573.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61691
+version: 1021.3.0
+---
+Retrieving the device context dashboards did not work as expected if the device had a space in its name. As a result, it was impossible to view such devices as the dashboard is by default the initial view for devices in the Cockpit application. This issue has been fixed and the device dashboard is always displayed properly.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-3-0-get-context-dashboards-fix-7570-graft-release-cd-7573.md]:
```suggestion
Retrieving the device context dashboards did not work as expected if the device had a space in its name. As a result, it was impossible to view such devices as the dashboard is by default the initial view for devices in the Cockpit application. This issue has been fixed and the device dashboard is always displayed.
```

File: content/change-logs/application-enablement/ui-c8y-1021-3-0-get-context-dashboards-fix-7570-graft-release-cd-7573.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fix retrieving device context dashboards
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-3-0-get-context-dashboards-fix-7570-graft-release-cd-7573.md]:
```suggestion
title: Device dashboards displayed for all devices
```

**Review by jdre-c8y — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1021-3-0-get-context-dashboards-fix-7570-graft-release-cd-7573.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61691
+version: 1021.3.0
+---
+Retrieving the device context dashboards did not work as expected if the device had a space in its name. As a result, it was impossible to view such devices as the dashboard is by default the initial view for devices in the Cockpit application. This issue has been fixed and the device dashboard is always displayed.
```
Comment (jdre-c8y) [inline content/change-logs/application-enablement/ui-c8y-1021-3-0-get-context-dashboards-fix-7570-graft-release-cd-7573.md:17]:
```suggestion
Retrieving the device context dashboards did not work as expected if the device had a space in its type. As a result, it was impossible to view such devices as the dashboard is by default the initial view for devices in the Cockpit application. This issue has been fixed and the device dashboard is always displayed.
```

**Pull request merged**